.helm/ecamp3/templates/browserless_deployment.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.browserless.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "browserless.name" . }}
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "app.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "app.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "app.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}-browserless
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.browserless.image.repository }}:{{ .Values.browserless.image.tag | default .Values.imageTag }}"
```

Comment:
Not sure if it makes sense to use `.Values.imageTag` as a fallback here... the browserless image will not be supplied by us anymore after this PR.

Reply:
Good point. Fixed in https://github.com/ecamp/ecamp3/pull/2656/commits/c0370bc8f32d9ed0a00f3ac156f33a012c39cf36